src/components/canvas-tools/templates/Line.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
+    <v-color-picker
+      v-model="lineColour"
+      :swatches="swatches"
+      mode="hexa"
+      show-swatches
+      hide-canvas
+      flat
+      hide-inputs
+      hide-mode-switch
+    />
+    <v-slider
+      prepend-icon="fa-ruler-vertical"
+      :hint="lineSizeHint"
+      v-model="lineSize"
+      :max="6"
+      :step="1"
+      ticks="always"
+    />
+    <v-radio-group v-model="lineEndStyle" row>
```

Comment:
can you group these together, put titles on the buttons and in general improve the UI of the different options for the line.
From your screenshot in discord you have 6 icons split into 2 rows of 3, can we ensure they are grouped nicely with a title and appropriate icons (or placeholders) for now. Work with Rites or myself for the UI if need be.

Reply:
can you show me a screenshot of what the tool settings look like?